From 6c8c984e55089fe0665b5a7833620e2bf25049a3 Mon Sep 17 00:00:00 2001
From: arter97 <qkrwngud825@gmail.com>
Date: Fri, 9 Jan 2015 15:43:07 +0900
Subject: [PATCH 01/13] Enable -Os back to audioflinger

Using plain -O3 causes compilation error.
Fix this by adding -Os to the CFLAGS.

Signed-off-by: arter97 <qkrwngud825@gmail.com>
---
 8962d10d6af6cb34655be20aa789645dc0ad6138.patch | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 8962d10d6af6cb34655be20aa789645dc0ad6138.patch

diff --git a/8962d10d6af6cb34655be20aa789645dc0ad6138.patch b/8962d10d6af6cb34655be20aa789645dc0ad6138.patch
new file mode 100644
index 0000000..9cb8189
--- /dev/null
+++ b/8962d10d6af6cb34655be20aa789645dc0ad6138.patch
@@ -0,0 +1,26 @@
+From 8962d10d6af6cb34655be20aa789645dc0ad6138 Mon Sep 17 00:00:00 2001
+From: arter97 <qkrwngud825@gmail.com>
+Date: Fri, 9 Jan 2015 15:43:07 +0900
+Subject: [PATCH] Enable -Os back to audioflinger
+
+Using plain -O3 causes compilation error.
+Fix this by adding -Os to the CFLAGS.
+
+Signed-off-by: arter97 <qkrwngud825@gmail.com>
+---
+ services/audioflinger/Android.mk | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/services/audioflinger/Android.mk b/services/audioflinger/Android.mk
+index 4eb13a3..1789c88 100644
+--- a/services/audioflinger/Android.mk
++++ b/services/audioflinger/Android.mk
+@@ -93,6 +93,8 @@ endif
+ endif
+ #QTI Resampler
+ 
++LOCAL_CFLAGS += -Os
++
+ LOCAL_MODULE:= libaudioflinger
+ LOCAL_32_BIT_ONLY := true
+ 
-- 
2.5.0

